CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Added
 
+- **CUMULUS-3298**
+  - Added extra time to replace launchpad token before it expires
```

Comment:
Nit: can we be more specific about what the functional changes are and why so that this clearly communicates the behavioral change and why it was needed?